### PR TITLE
Fix bug: can't play video in obbfile.

### PR DESCRIPTION
**Pull request by halx99**

### CODE REVIEW

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java [hunk 1]
```diff
@@ -307,7 +307,11 @@ private void openVideo() {
             mDuration = -1;
             mCurrentBufferPercentage = 0;
             if (mIsAssetRouse) {
-                AssetFileDescriptor afd = mCocos2dxActivity.getAssets().openFd(mVideoFilePath);
+                AssetFileDescriptor afd = null;
+                if (Cocos2dxHelper.getObbFile() != null) {
+                    afd = Cocos2dxHelper.getObbFile() .getAssetFileDescriptor(mVideoFilePath);
+                }
+                if(afd == null) afd = mCocos2dxActivity.getAssets().openFd(mVideoFilePath);
```
Comment (minggo) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java]: Why check OBB file first? Is using OBB file more common?

Comment (halx99) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java]: It doesn't matter what order check.

Comment (minggo) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java]: But check assets may make it more faster if video files are more commonly stored in assets.

Comment (halx99) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java]: yes, you are right.

Comment (halx99) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java]: @minggo I change the check order, please review

**Review by minggo — APPROVED** [review]

**Comment by minggo** [comment]
it is synchronized to metal-support branch in https://github.com/cocos2d/cocos2d-x/pull/19586

**Pull request merged**